src/auslib/web/admin/base.py

```diff
@@ -88,7 +88,7 @@ def setup_request():
 
 @app.after_request
 def complete_request(response):
-    if request.full_path.startswith("/v2"):
+    if hasattr(request, "transaction"):
```

Comment:
I wonder if it's worth keeping the `full_path` check, for symmetry, but maybe not.

Reply:
I think the hasattr way is better: We know it's possible to have "/v2" and not have request.transaction; throwing an AttributeError here doesn't help us at all.